React/Fabric/RCTConversions.h

```diff
@@ -35,32 +35,32 @@ inline std::string RCTStringFromNSString(NSString *string)
   return std::string{string.UTF8String ?: ""};
 }
 
-inline UIColor *_Nullable RCTUIColorFromSharedColor(facebook::react::SharedColor const &sharedColor)
+inline RCTUIColor *_Nullable RCTUIColorFromSharedColor(facebook::react::SharedColor const &sharedColor) // TODO(macOS GH#774)
 {
   if (!sharedColor) {
     return nil;
   }
 
   if (*facebook::react::clearColor() == *sharedColor) {
-    return [UIColor clearColor];
+    return [RCTUIColor clearColor]; // TODO(macOS GH#774)
   }
 
   if (*facebook::react::blackColor() == *sharedColor) {
-    return [UIColor blackColor];
+    return [RCTUIColor blackColor]; // TODO(macOS GH#774)
   }
 
   if (*facebook::react::whiteColor() == *sharedColor) {
-    return [UIColor whiteColor];
+    return [RCTUIColor whiteColor]; // TODO(macOS GH#774)
   }
 
   auto components = facebook::react::colorComponentsFromColor(sharedColor);
-  return [UIColor colorWithRed:components.red green:components.green blue:components.blue alpha:components.alpha];
+  return [RCTUIColor colorWithRed:components.red green:components.green blue:components.blue alpha:components.alpha]; // TODO(macOS GH#774)
 }
 
 inline CF_RETURNS_RETAINED CGColorRef
 RCTCreateCGColorRefFromSharedColor(const facebook::react::SharedColor &sharedColor)
 {
-  return CGColorRetain(RCTUIColorFromSharedColor(sharedColor).CGColor);
+  return CGColorRetain(RCTUIColorFromSharedColor(sharedColor).CGColor); // TODO(macOS GH#774)
```

Comment:
nit: We don't really need this TODO tag. The way we use `RCTUIColorFromSharedColor` isn't changing; the only difference is the written return value, which we already covered on line 38 of this file.